jenkins/databricks/build.sh

```diff
@@ -41,6 +41,7 @@ rm -rf spark-rapids
 mkdir spark-rapids
 tar -zxvf $SPARKTGZ -C spark-rapids
 cd spark-rapids
+export WORKSPACE=`pwd`
```

Comment:
Define `WORKSPACE` in the DB cluster to fetching cudf/spark version info:

https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/build/dependency-info.sh#L30